Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'rubocop', '< 0.42.0'
 
 gem 'json_pure', '< 2.0.2'
 gem 'net-http-persistent', '<= 2.9.4', :require => false if RUBY_VERSION < '2.0.0'
+gem 'addressable', '<= 2.4.0', :require => false if RUBY_VERSION < '2.0.0'
```

Comment:
To fix spec errors as the latest adressable (2.5.0) needs ruby 2.0 and then the 1.9 integration tests on Travis failes.